metastore/scripts/upgrade/hive/hive-schema-3.0.0.hive.sql

```diff
@@ -0,0 +1,1334 @@
+-- HIVE system db
+
+CREATE DATABASE SYS;
```

Comment:
Yes, I think you understood the question right. You created the tables (say BUCKETING_COLS) in db sys, they could work like a "view" since they do not have the actual data but you have to use JDBCStoragerHandler to retrieve the data from their underlying table. It introduces the feature dependency on JDBCStorageHandler and also the query processing occurs at Hive site. In addition, it looks like only the MySql JDBCStorageHandler are supported in Hive at this moment. My question was that why did not you create pure views in sys without using JDBCStoragerHandler (just like you did for information_schema db)? To me, it seems more straightforward and less maintenance. Or did I miss something here. Thanks